corehq/apps/case_search/utils.py

```diff
@@ -59,12 +61,30 @@ def _get_initial_search_es(self):
         return search_es
 
     def _assemble_optional_search_params(self):
+        self._validate_multiple_parameter_values()
         self._add_xpath_query()
         self._add_owner_id()
         self._add_blacklisted_owner_ids()
         self._add_daterange_queries()
         self._add_case_property_queries()
 
+    def _validate_multiple_parameter_values(self):
+        disallowed_multiple_value_parameters = [
+            CASE_SEARCH_BLACKLISTED_OWNER_ID_KEY,
+            'owner_id',
+            CASE_SEARCH_XPATH_QUERY_KEY,
+        ]
+
+        for key, val in self.criteria.items():
+            if not isinstance(val, list):
+                continue
+            daterange_invalid = any([v.startswith('__range__') for v in val])
+            if key in disallowed_multiple_value_parameters or '/' in key or daterange_invalid:
+                raise CaseFilterError(
+                    _("Multiple values are only supported for simple text lookups"),
```

Comment:
nit: rename var and slight rewording of error message
```suggestion
            is_daterange = any([v.startswith('__range__') for v in val])
            if key in disallowed_multiple_value_parameters or '/' in key or is_daterange:
                raise CaseFilterError(
                    _("Multiple values are only supported for simple text and range searches"),
```